layers/meta-resin-edison/recipes-connectivity/modemmanager/files/77-mm-tty-device-blacklist.rules

```diff
@@ -0,0 +1,3 @@
+ACTION!="add|change", GOTO="mm_tty_blacklist_end"
```

Comment:
I'm a little confused by this rule. So for add and change you skip the ID ignore variable set. So you only ignore it for action remove?

Reply:
For an "add" or a "change" event we add a "ID_MM_DEVICE_IGNORE" so that ModemManager does not handle these devices

Reply:
Uh. I missed the "!".